Formula/qt5.rb

```diff
@@ -90,12 +100,32 @@ def install
       -qt-pcre
       -nomake tests
       -no-rpath
+      -pch
+      -system-proxies
+      -optimized-tools
```

Comment:
-pch is never default since qt 3.x,
-optimized-tools is only on when building release build, so, I think it's ok to omit.